cookbooks/free_surface/doc/free_surface.md

```diff
@@ -69,9 +69,8 @@ After running the cookbook, you may modify it in a number of ways:
 
 -   Try a model in a different geometry, such as spherical shells.
 
-<img src="cookbooks/free_surface/doc/free_surface_blob.png" title="fig:" id="fig:freesurface" style="height:25.0%" alt="Evolution of surface topography due to a rising blob. On the left is a snapshot of the model setup. The right shows the value of the highest topography in the domain over 18 Myr of model time. The topography peaks at 167 meters after 5.5 Myr. This cookbook may be run with the cookbooks/free_surface/free_surface.prm input file." />
-<img src="cookbooks/free_surface/doc/free_surface_topography.png" title="fig:" id="fig:freesurface" style="height:25.0%" alt="Evolution of surface topography due to a rising blob. On the left is a snapshot of the model setup. The right shows the value of the highest topography in the domain over 18 Myr of model time. The topography peaks at 167 meters after 5.5 Myr. This cookbook may be run with the cookbooks/free_surface/free_surface.prm input file." />
+```{figure-md} fig:freesurface
+<img src="free_surface_multi.*" alt="Figure"  width="100%"/>
```

Comment:
No need to add `multi` to all figures. Think about how this will look like when we are done removing the pdf manual. Then this will be the only figure, so no need to call it `multi`.
```suggestion
<img src="free_surface.*" alt="Figure"  width="100%"/>
```